ibm/service/power/ibm_pi_constants.go

```diff
@@ -383,8 +384,11 @@ const (
 	StatusError   = "ERROR"
 	StatusBuild   = "BUILD"
 	StatusPending = "PENDING"
+	SctionAccess  = "access"
 	SctionStart   = "start"
+	SctionSource  = "source"
 	SctionStop    = "stop"
+	SctionReset   = "reset"
```

Comment:
This a typo -- should have been ActionAccess ...
And move it up to after 
Arg
Attr
Status/State